styles/main.css

```diff
@@ -1,166 +1,174 @@
-    align-items: center;
-    padding-top: 50px; /* add this line to adjust the padding on the top of the main section */
-    margin-top: 100px; /* change this value to adjust the margin between the header and the image */
-  }
-  
-  img {
-    /*
-    max-width: 50%;
-    display: block;
-    margin: auto;
-    */
-    width: 400px;
-    height: auto;
-    display: block;
-    margin: auto;
+}
 
+/* For screens larger than 900px */
+@media (min-width: 900px) {
+  header {
+    font-size: 48px;
+    height: 100px; /* change this value to adjust the height of the header for large screens */
   }
-  
-  .external-links {
-    margin-top: 20px;
-    display: flex;
-    justify-content: center;
-    align-items: center; /* add this line to center the external links horizontally */
-  }
-  
-  .external-links a {
-    margin: 0 10px;
-    text-decoration: none; /* add this line to remove the underline from the links */
-    color: black; /* add this line to change the color of the links */
-  }
-  
-  /* Media queries */
-  
-  /* For screens smaller than 600px */
-  @media (max-width: 600px) {
-    header {
-      font-size: 24px;
-      height: 60px; /* change this value to adjust the height of the header for small screens */
-    }
-  
-    nav li {
-      margin: 0 5px;
-    }
-  }
-  
-  /* For screens between 600px and 900px */
-  @media (min-width: 600px) and (max-width: 900px) {
-    header {
-      font-size: 36px;
-    }
-  
-    nav li {
-      margin: 0 10px;
-    }
-  }
-  
-  /* For screens larger than 900px */
-  @media (min-width: 900px) {
-    header {
-      font-size: 48px;
-      height: 100px; /* change this value to adjust the height of the header for large screens */
-    }
-  }
+}
 
-  /* WRITING PAGE */
+/* WRITING PAGE */
 
-  .columns-container {
-    display: flex;
-    justify-content: space-between;
-    width: 100%;
-  }
-  
-  .column-left,
-  .column-right {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    flex: 1;
-    padding: 20px;
-  }
-  
-  .column-left {
-    border-right: 1px solid black;
-  }
-  
-  h2 {
-    font-weight: bold;
-    color: black;
-    margin-bottom: 10px;
-    text-align: left;
-  }
-  
-  ul {
-    list-style: none;
-    margin: 0;
-    padding: 0;
-  }
-  
-  li {
-    margin-bottom: 10px;
-  }
-  
-  a {
-    color: black;
-    text-decoration: none;
-  }
- /*
-  #essay pre {
-    font-family: Lato;
-    font-size: 16px;
-    line-height: 1.5;
-    color: #333;
-    width: 80%;
-    margin: 20px 0;
-  }
-*/
-  
\ No newline at end of file
+.columns-container {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.column-left,
+.column-right {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex: 1;
+  padding: 20px;
+}
+
+.column-left {
+  border-right: 1px solid black;
+}
+
+.essay {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: space-between;
+  width: 68%;
```

Comment:
same thing for this file, most of the changes were just indentation. i think I did an `align-content: center;` and made the width a bit smaller, and maybe some other small things here or there but very few changes overall